docs/develop/Sample_Data.md

```diff
@@ -33,32 +33,7 @@ Cast ballots will be tallied and spoiled ballots are decrypted.
 
 ### Election Record
 
-The Election Record are the files required at the end of the election to verify the election and are intended to be posted publicly. There should not be any sensitive data (eg: no guardian private keys) in the record. These include the following:
-
-- Manifest
-- Election Context
-- Election Constants
-- Encrypted Tally
-- Decrypted Tally
-- Guardian Records
-- Encryption Device Information
-- Encrypted Ballots
-- Decrypted Spoiled Ballots
-
-**Example Folder Structure:**
-
-```
-📂 artifacts
---- 📄 manifest.json
---- 📄 context.json
---- 📄 constants.json
---- 📄 encrypted_tally.json
---- 📄 tally.json
---- 📁 guardians
---- 📁 devices
---- 📁 encrypted_ballots
---- 📁 spoiled_ballots
-```
+The Election Record are the files required at the end of the election to verify the election and are intended to be posted publicly. There should not be any sensitive data (eg: no guardian private keys) in the record. 
```

Comment:
"record are" has pluralization mismatch "is the group of files" is better
"and is intended"
"There should not be any" <- "It should not contain any"